src/dpr/components/copy-url-button/clientClass.mjs

```diff
@@ -0,0 +1,17 @@
+import { DprClientClass } from '../../DprClientClass.mjs'
+
+export default class CopyUrlButton extends DprClientClass {
+  static getModuleName() {
+    return 'copy-url-button'
+  }
+
+  initialise() {
```

Comment:
Do you need to do a `document.querySelectorAll` here? The getElement() function would return the div around the container for the button. You can set up just this one button.

As it is currently, I think it will initialise all Copy URL buttons for every Copy URL button there is (3 buttons would run 9 initialisations in total).